runtime/android/core_internal/src/org/xwalk/core/internal/ReflectionHelper.java

```diff
@@ -2,16 +2,53 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/*
```

Comment:
(Note for security reviewers coming to look at this code before it gets merged: Francesco is attempting to clear this with the appropriate person but has not heard back yet)